pkgs/top-level/all-packages.nix

```diff
@@ -5438,6 +5438,8 @@ with pkgs;
 
   n2n = callPackage ../tools/networking/n2n { };
 
+  nest-cli = callPackage ../development/tools/admin/nest-cli { };
```

Comment:
The directory was put in `pkgs/development/tools/nest-cli`, not `pkgs/development/tools/admin/nest-cli`. Which one were you intending?

Reply:
I had meant the directory I put it in, just got confused when including it in `all-packages.nix` 😄